plugins/mergeStyles.js

```diff
@@ -0,0 +1,71 @@
+'use strict';
+
+exports.type = 'full';
+exports.active = true;
+exports.description = 'merge multiple style elements into one';
+
+const cssTools = require('../lib/css-tools'),
+  xast = require('../lib/xast');
```

Comment:
Add own const for each variable

Reply:
Also import each function separately like named imports

Reply:
OK.